src/libraries/System.Runtime/tests/System.Runtime.Tests.csproj

```diff
@@ -285,6 +285,23 @@
     <Compile Include="System\ExitCodeTests.Unix.cs" />
   </ItemGroup>
 
+  <!--
+    These tests are currently not R2R-Compatible, so we disable them whenever
+    we are using the TestReadyToRun flag.
+  -->
+  <ItemGroup Condition="'$(TestReadyToRun)' == 'true'">
+    <Compile Remove="System\Runtime\JitInfoTests.cs" />
+    <Compile Remove="System\Type\TypeTests.Get.cs" />
+  </ItemGroup>
+
+  <!--
+    This test surprisingly only fails on Linux and Windows when TestReadyToRun
+    is present. OSX runs just fine.
```

Comment:
Can we create a GitHub tracking issue and mention it in the comment? Other than that, I was under the impression that library tests support the attribute [ActiveIssue], would it be possible to mark the problematic tests with this attribute rather than tweaking the project files?